v2/stacks/feedback/default.env

```diff
@@ -0,0 +1,15 @@
+# -- Global variable overrides --
+IS_PUBLISHING="false"
+MAIL_ENCRYPTION=${MAIL_ENCRYPTION:-false}
+ZEBEDEE_URL=http://http-echo:5678
```

Comment:
Add a comment here to explain we're stubbing the health check.

Reply:
Done